src/style/Card.scss

```diff
@@ -0,0 +1,31 @@
+.container {
+    display: flex;
+    width: 300px;
```

Comment:
I suggest you to add an optional prop size (small, normal, large) set by default to "normal => 300px" which will have 3 different sizes :)